config/blazer.yml

```diff
@@ -0,0 +1,44 @@
+# see https://github.com/ankane/blazer for more info
+
+data_sources:
+  main:
+    url: <%= ENV["BLAZER_DATABASE_URL"] %>
+
+    # statement timeout, in seconds
```

Comment:
hey there is a lot of commented code in this gem setup (in this file and others in this PR) - can we remove? easier to see what is being set up without all of the surrounding commented code